doc/developer_guide/transformations.rst

```diff
@@ -55,43 +55,21 @@ Transformations
 Kernel Transformations
 ======================
 
-PSyclone is able to perform kernel transformations. Currently it has
-two ways to apply transformations: by directly manipulating the
-language AST or by translating the language AST to PSyIR, applying the
-transformation in the PSyIR and using one of the back-ends to generate
-the resulting code.
-
-For now, both methods only support the fparser2 AST for kernel code.
-This AST is obtained by converting the fparser1 AST (stored
-when the kernel code was originally parsed to process the meta-data)
-back into a Fortran string and then parsing that with fparser2.
-(Note that in future we intend to adopt fparser2 throughout PSyclone so that
-this translation between ASTs will be unnecessary.)
-The `ast` property of the `psyclone.psyGen.Kern` class is responsible
-for performing this translation the first time it is called. It also
-stores the resulting AST in `Kern._fp2_ast` for return by future calls.
-
-Alternatively, one can call the `psyclone.psyGen.CodedKern.get_kernel_schedule()`
-to generate the PSyIR representation of the kernel code. 
+PSyclone is able to perform kernel transformations by obtaining the PSyIR
+representation of the kernel with:
 
```

Comment:
This is great but the docstring for `get_kernel_schedule` still talks about the fp2 AST. Please could you update it.

Reply:
Done